lib/parallel_tests/test/runner.rb

```diff
@@ -78,7 +78,7 @@ def execute_command(cmd, process_number, num_processes, options)
           cmd = "nice #{cmd}" if options[:nice]
           cmd = "#{cmd} 2>&1" if options[:combine_stderr]
 
-          puts cmd if options[:verbose]
+          puts cmd if options[:verbose] && !options[:quiet]
```

Comment:
giving quiet and verbose should be a bug, would rather not add extra code for that